deploy/newChain/2_agTokenImplementation.ts

```diff
@@ -9,16 +10,8 @@ const func: DeployFunction = async ({ deployments, ethers, network }) => {
   const { deployer } = await ethers.getNamedSigners();
   const stableName = 'EUR';
 
-  let implementationName: string;
-  let proxyAdmin: string;
-
-  if (network.config.chainId == 1 || !network.live) {
-    implementationName = 'AgTokenSideChain';
-    proxyAdmin = registry(ChainId.MAINNET)?.ProxyAdmin!;
-  } else {
-    implementationName = 'AgTokenSideChainMultiBridge';
-    proxyAdmin = (await deployments.get('ProxyAdmin')).address;
-  }
+  const implementationName = 'AgTokenSideChainMultiBridge';
```

Comment:
We can also rename the contract `AgTokenSideChainMultiBridge` without sidechain anymore as they are all identical

Reply:
So AgTokenMultiBridge? Because the L2s agEUR are different from the L1 agEUR